hima890/AirBnB_clone
# Add static HTML and CSS for AirBnB project

### **This pull request introduces the static HTML and CSS files for the AirBnB project. The changes include the main structure of the webpage with improved accessibility and responsive design.**

**Changes Made:**
![webApp1](https://github.com/hima890/AirBnB_clone/assets/48498413/448af0f5-57cb-4288-8df6-6f189a2de54a)
![webApp2](https://github.com/hima890/AirBnB_clone/assets/48498413/3a9be3f8-c096-41a2-85cc-e5345b9387d0)

**HTML Files:**

Added index.html with the structure for header, main content (filters and places sections), and footer.
Included ARIA attributes for better accessibility.
Ensured proper semantic HTML elements for improved readability and maintainability.
**CSS Files:**

Added styles/4-common.css for common styles.
Added styles/3-header.css for header-specific styles.
Added styles/3-footer.css for footer-specific styles.
Added styles/6-filters.css for filters section styles.
Added styles/100-places.css for places section styles.
Ensured responsive design using media queries.
Included accessibility improvements like better focus states and ARIA roles.
**Files Added:**

**index.html**
styles/4-common.css
styles/3-header.css
styles/3-footer.css
styles/6-filters.css
styles/100-places.css

**How to Test:**
Pull the branch and open index.html in a browser.
Verify the layout and styling of the header, filters, places, and footer sections.
Resize the browser window to ensure responsiveness at various breakpoints (e.g., 320px, 768px, 1024px).
Use a screen reader to check the accessibility features like ARIA roles and labels.

**Screenshots:**
(Include before and after screenshots of the webpage if applicable)

**Additional Notes:**

The static files serve as the foundation for the dynamic content to be integrated later.
Ensure all images referenced in the CSS are placed in the appropriate directory (images).
